examples/connext_dds/group_coherent_presentation/py/GroupCoherentExample_subscriber.py

```diff
@@ -0,0 +1,168 @@
+#
+# (c) 2022 Copyright, Real-Time Innovations, Inc.  All rights reserved.
+#
+# RTI grants Licensee a license to use, modify, compile, and create derivative
+# works of the Software solely for use with RTI products.  The Software is
+# provided "as is", with no warranty of any type, including any warranty for
+# fitness for any purpose. RTI is under no obligation to maintain or support
+# the Software.  RTI shall not be liable for any incidental or consequential
+# damages arising out of the use or inability to use the software.
+#
+import argparse
+
+import rti.connextdds as dds
+from GroupCoherentExampleType import AlarmCode, Alarm, HeartRate, Temperature
+
+
+def process_data(reader):
+
+    # Take all samples.  Samples are loaned to application, loan is
+    # returned when LoanedSamples destructor called or when you return
+    # the loan explicitly.
+    samples_read = 0
+    samples = reader.take()
+
+    for (data, info) in samples:
+        if info.valid:
+            print("Received: {}".format(data))
```

Comment:
```suggestion
            print(f"Received: {data}")
```

Prefer `f`-strings for formatting